project.mml

```diff
@@ -1969,6 +1971,7 @@ Layer:
               'leisure_' || CASE WHEN leisure IN ('nature_reserve') THEN leisure ELSE NULL END
             ) AS feature,
             name,
+            tags->'information' as information,
```

Comment:
What's this line for in `text-poly-low-zoom` layer? I have tested that it works without this code as well on all the zoom levels.

Reply:
Same thing like #3196. :)

Reply:
Oh, boy, not again... :smile:

Did you have a problem without this code or it's just in case? Did you try with updated Docker image (I mean with a new Kosmtik :smile: ).

I'm ready to merge this code without this line any time. Especially since production rendering servers have no such problem and there are no other mid/high zoom features that I'm aware of which need such addition. Would you like to split it, so we could discuss the low zoom mystery separately?

Reply:
I'll do some tests again when I have time.